modules/aws-exfiltration-protection/variables.tf

```diff
@@ -3,60 +3,90 @@ variable "databricks_account_id" {
   description = "Databricks Account ID"
 }
 
-variable "tags" {
-  default     = {}
-  type        = map(string)
-  description = "Optional tags to add to created resources"
+resource "random_string" "naming" {
+  special = false
+  upper   = false
+  length  = 6
```

Comment:
Usually `variables.tf` is used only for variables declaration...

Reply:
corrected